tests/test_data.py

```diff
@@ -3156,23 +3156,3 @@ def test_export_bldg_threshold(self):
         prj.buildings[-1].thermal_zones[-1].use_conditions.with_ideal_thresholds = True
         prj.calc_all_buildings()
         prj.export_aixlib()
-
-    def test_tz_naming(self):
-
-        from teaser.logic.buildingobjects.building import Building
-        from teaser.logic.buildingobjects.thermalzone import ThermalZone
-
-        # warnings for not calculated buildings
-        bld = Building(parent=prj)
-        tz1 = ThermalZone(parent=bld)
-        tz1.name = "living"
-
-        tz2 = ThermalZone(parent=bld)
-        tz2.name = "kitchen"
-
-        tz3 = ThermalZone(parent=bld)
-        tz3.name = "living"
-
-        assert tz1.name == "living"
-        assert tz2.name == "kitchen"
-        assert tz3.name == "living_1"
```

Comment:
why did you delete this test?

Reply:
Not intendet, this test does not exist in the master, maybe a merge conflict. It should be in there.

Reply:
Ok, I commented the test now as the code that is needed for test is only in dev and not in this branch. Don't complety understand what went wrong here. But lets merge this into dev after the tests completed and then merge dev -> master. We need to uncomment this test then again by hand before the merge into master.